Clients/GUI/Pages/SignUp.razor

```diff
@@ -0,0 +1,59 @@
+﻿@page "/signup"
+@using GUIModels
+@using GUI.Scripts
+@inject NavigationManager NavigationManager
+
+    <div class="col-12">
```

Comment:
These names of classes are automatical? Can we change them to more understandable?

Reply:
These are bootstrap styles